src/winuwp/runner_uwp/CMakeLists.txt

```diff
@@ -0,0 +1,127 @@
+﻿cmake_minimum_required (VERSION 3.8)
+set(CMAKE_SYSTEM_NAME WindowsStore)
+set(CMAKE_SYSTEM_VERSION 10.0)
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED YES)
+
+include(CMakePrintHelpers)
+
+project (runner LANGUAGES CXX)
```

Comment:
Should this be:
```
project({{projectName}} LANGUAGES CXX)
```

Reply:
Actually, I think it's fine to do that cleanup in a followup.

Reply:
is this templating or cmake syntax?